compose/web/benchmark/build.gradle

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import androidx.build.AndroidXUiPlugin
+import androidx.build.LibraryGroups
+import androidx.build.Publish
+
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath "com.github.node-gradle:gradle-node-plugin:3.0.1"
+    }
+}
+
+plugins {
+    id("AndroidXPlugin")
```

Comment:
Do we really need AndroidXPlugin and ANdroidXUiPlugin in this particular module? What they actualy do?

Reply:
it looks like we don't need AndroidXPlugin.
As for AndroidXUiPlugin, it currently enables some compose-js specific flag: 
```
generateDecoys=true
```
this flag is needed for every module where we use compose with js.
But we can think about moving the flag somewhere else.